### PR TITLE
Docs 12717 mongodb settings

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
Small update to the MongoDB page to include the option to use a connection string.

Also piggybacking an updated screenshot for BigQuery to capture the recently updated file upload button.

### CODE REVIEW

File: docs/administration-guide/databases/mongodb.md [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
-This guide covers:
-
+ - Using a connection string.
  - General connectivity concerns.
  - Connecting to a MongoDB Atlas cluster.
 
+## Using a connection string
+
+If you'd prefer to connect to MongoDB using a [connection string](https://docs.mongodb.com/manual/reference/connection-string/),
+click on **Paste a connection string**. The Metabase UI will update with a field to paste your connection string, as well as present the following options:
```
Comment (robdaemon) [inline docs/administration-guide/databases/mongodb.md:12]: I think we may need to add a comment about the following connection string parameters NOT being supported at this time:

- `tlsCertificateKeyFile`
- `tlsCertificateKeyFilePassword`
- `tlsCAFile`

File: docs/administration-guide/databases/mongodb.md [hunk 2]
```diff
@@ -1,27 +1,36 @@
...
+ - **Connect using `DNS SRV`**, which is the recommended method for newer Atlas clusters.
 
- - Have you checked your cluster host whitelist?  When testing a connection but seeing failure, have you tried setting the IP whitelist to `0.0.0.0/0`?  This allows connections from any IP addresses. If you know the IP address(es) or CIDR block of clients, use that instead.
+ - **Have you checked your cluster host whitelist?** When testing a connection but seeing failure, have you tried setting the IP whitelist to `0.0.0.0/0`?  Whitelisting this address allows connections from any IP addresses. If you know the IP address(es) or CIDR block of clients, use that instead.
 
- - Are you using self-signed certificates?  While Metabase doesn't support providing a self-signed cert through the Metabase UI, you can still use the command line to configure this.  Just copy the existing store and add the self-signed cert to it like this:
+ - **Are you using self-signed certificates?** While Metabase doesn't support providing a self-signed cert through the Metabase UI, you can still use the command line to configure this. Just copy the existing store and add the self-signed cert:
```
Comment (robdaemon) [inline docs/administration-guide/databases/mongodb.md:24]: This section needs to be changed as a result of #12564 - we now support providing the mongo cert in the UI (but only when using the individual fields view, not the connection string view)

**Review by robdaemon — CHANGES_REQUESTED** [review]

File: docs/administration-guide/databases/mongodb.md (hunk 2 above)
Comment (jeff-bruemmer) [reply docs/administration-guide/databases/mongodb.md]: Thanks for the notes, @robdaemon. I rearranged the doc and added some navigation. Called out the fact that you can set up SSL via the default method and NOT via a connection string. I also carved out adding a cert via the command line as its own section.

**Review by mazameli — APPROVED** [review]
This seems good to go to me provided @robdaemon's happy with the changes he requested.

**Review by robdaemon — APPROVED** [review]
:+1:

**Pull request merged**